tests/integration/targets/nd_compliance_analysis/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# Test code for the ND modules
+# Copyright: (c) 2022, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an Nexus Dashboard host, username and password
+  fail:
+    msg: 'Please define the following variables: nd_hostname, nd_username, nd_password, insights_group, and site_name.'
```

Comment:
Why are we adding nd_hostname, nd_username, nd_password?
Those are not necessary as they are taken from ansible_user from the inventory directly.